modules/is_master/manifests/custom.pp

```diff
@@ -0,0 +1,20 @@
+#----------------------------------------------------------------------------
+#  Copyright (c) 2018 WSO2, Inc. http://www.wso2.org
```

Comment:
Shall we update the Copyright notice here?

Reply:
https://github.com/wso2/puppet-is/pull/58/commits/b8ba5cf3de87228146971ff913f98bd16021353b